File: Media2.py
#Define a variável (s) com o valor de 0
s = 0

#Cria um loop com um raio de 10
for i in range(10):

    #Diz que se i for menor que 1, vai pedir um número
    if i < 1:

        #Declara (n1) e entrega para ele o valor do input(entrada) e a mensagem
        n1 = int(input("Digite um número: "))

        #Declara o valor de (s) e faz com que o valor dele some e atualize com (n1)
        s += n1

    #Diz que se i não for menor q 1 ele pede outro número, soma ele e divide por 2
    else:

        
        n1 = int(input("Digite outro número: "))
        s += n1 
        s /= 2

        #Atualiza o valor da média (s) na tela
        print(f'A média de agora é {s}')


#Mostra a média final na tela
print(f"A média é {s}")

File: Media2.c
//Puxa a Biblioteca do C para mostrar informação na tela
#include <stdio.h>

int main() {
    //Declara a variável "sm" e define ela com valor 0 (float = Número com vírgula) 
    float sm = 0;
    //Abre um loop que vai pedir 10 números para fazer a média de 2 em 2 somando-os
    for (int i = 0; i < 10; i++)
    {
        //Declara a Variável (n)
        float n;
        //Diz: se i for menor que 1 ele vai pedir um número
        if (i < 1)
        {
            //Mostra a mesnsagem
            printf("Digite um número:\n");
            //Pede um número
            scanf("%f", &n);
            //Declara (sm) e faz com que o valor dele some e atualize com (n)
            sm += n;
        }
        //Diz que se i não for menor q 1 ele pede outro número,soma ele e divide por 2
        else
        {
            //Mostra a mensagem
            printf("Digite outro número: \n");
            //Pede outro número
            scanf("%f", &n);
            //Declara que (sm) é igual a soma do "sm + n" e divide por 2
            sm = (sm + n) / 2;
        }
        //A cada número é mostrado na tela a nova média
        printf("A média é: %.1f\n", sm);
    
    }
return 0;

    
}